src/index.ts

```diff
@@ -8,7 +8,7 @@ async function run(): Promise<void> {
 
     // Get inputs
     // https://docs.github.com/en/rest/releases/releases#create-a-release
-    const tagName = core.getInput('tag_name', { required: false }) || github.context.ref.replace('refs/tags/', '')
+    const tagName = github.context.ref.replace('refs/tags/', '')
```

Comment:
The tag_name is an optional inputs with an empty string as defaults .

In a case, you can specify a tagName you like rather than from context.

Reply:
I can't totally agree. In general, the trigger condition is a special signature. 

```yml
on:
  push:
    tags:
      - v*
```

Why change the release name when you've tagged it?

Reply:
Oh my ~, I got you gangster. Holy mf. 😭

Reply:
Please re-create your PR.

Reply:
#12